### PR TITLE
:bulb: Upate Benchmark build script and deployment files

### PR DESCRIPTION
**Pull request by haitwang-cloud**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR optimizes the build scripts to improve the build process's efficiency and reliability. It also refines the Kubernetes deployment files to enhance the service's stability and performance in a Kubernetes environment. Both these modifications are crucial for maintaining a smooth development workflow and ensuring robust application deployment.

**Which issue(s) this PR fixes**:
This PR addresses issues related to inefficient build processes and non-optimal Kubernetes deployment configurations.

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:

No, this PR does not introduce any user-facing changes. The optimizations are related to the build process and Kubernetes deployment configurations, which are typically behind-the-scenes changes that should not directly impact the end-user experience.

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Benchmarking the vGPU scheduler
+
+## Prerequisites
+
+### how to build the benchmark image
+
+```bash
+cd k8s-vgpu-scheduler/benchmarks/ai-benchmark
```
Comment (wawa0210) [inline benchmarks/README.md]: `k8s-vgpu-scheduler/benchmarks/ai-benchmark` --> `HAMi/benchmarks/ai-benchmark`

File: benchmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+## How to install the official nvidia device plugin
+
+Please refer to  [Quick Start](https://github.com/NVIDIA/k8s-device-plugin?tab=readme-ov-file#quick-start) in the official nvidia device plugin repository.
+
+## Run the benchmark
+
+```bash
+cd k8s-vgpu-scheduler/deployments
```
Comment (wawa0210) [inline benchmarks/README.md]: the same as before

File: benchmarks/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+k apply -f job-on-hami.yml
```
Comment (wawa0210) [inline benchmarks/README.md]: `k` --> `kubectl`,This alias should only take effect in your local environment

File: benchmarks/ai-benchmark/Dockerfile [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 WORKDIR ai-benchmark
-RUN pip install -e .
 
-ENTRYPOINT [ "python", "bin/ai-benchmark.py" ]
+# Update the package list and install git and apt-utils
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends apt-utils git && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install --no-cache-dir --upgrade pip && \
+    git clone https://github.com/haitwang-cloud/ai-benchmark . && \
```
Comment (wawa0210) [inline benchmarks/ai-benchmark/Dockerfile]: Can you explain why you use your own repo? Don't know the context here

**Comment by wawa0210** [comment]
@haitwang-cloud Thank you very much for your contribution. Can you please attend our weekly meeting this Friday and introduce your PR in detail? This can speed up the review process

File: benchmarks/ai-benchmark/Dockerfile (hunk 4 above)
Comment (haitwang-cloud) [reply benchmarks/ai-benchmark/Dockerfile]: Hi there, it seems that the previous implementation isn't functioning as expected in my tests, which I believe is due to a lack of maintenance. As a result, I've taken the initiative to make some enhancements myself. I'd be more than happy to contribute these improvements back to the Project-HAMi.

**Comment by wawa0210** [comment]
> > @haitwang-cloud Thank you very much for your contribution. Can you please attend our weekly meeting this Friday and introduce your PR in detail? This can speed up the review process
> > the weekly meeting information -> https://github.com/Project-HAMi/HAMi?tab=readme-ov-file#meeting--contact
> 
> Tks for that, I will attend the meeting.

I'm very sorry, we have rescheduled the weekly meeting on Friday~ Can we discuss this issue privately?

**Comment by haitwang-cloud** [comment]
> > > @haitwang-cloud Thank you very much for your contribution. Can you please attend our weekly meeting this Friday and introduce your PR in detail? This can speed up the review process
> > > the weekly meeting information -> https://github.com/Project-HAMi/HAMi?tab=readme-ov-file#meeting--contact
> > 
> > 
> > Tks for that, I will attend the meeting.
> 
> I'm very sorry, we have rescheduled the weekly meeting on Friday~ Can we discuss this issue privately?

Sure, no worries, plz guide me what to do next.

**Comment by wawa0210** [comment]
> > > > @haitwang-cloud Thank you very much for your contribution. Can you please attend our weekly meeting this Friday and introduce your PR in detail? This can speed up the review process
> > > > the weekly meeting information -> https://github.com/Project-HAMi/HAMi?tab=readme-ov-file#meeting--contact
> > > 
> > > 
> > > Tks for that, I will attend the meeting.
> > 
> > 
> > I'm very sorry, we have rescheduled the weekly meeting on Friday~ Can we discuss this issue privately?
> 
> Sure, no worries, plz guide me what to do next.

Are you in hami's slack channel? Can you join it and let's schedule a meeting again?

**Comment by haitwang-cloud** [comment]
> hi～could you attend meeting at 4pm today? i'll add this to meeting agenda
Sure， but I can only joint the meeting at 4:30 PM cause I have another meeting today.

**Comment by haitwang-cloud** [comment]
Hey @wawa0210, could you please take a look at this again? I've updated the personal link to the HAMi repo link.

**Comment by haitwang-cloud** [comment]
Hey @wawa0210 @archlitchi  , could you please take a look at this again? I've updated the personal link to the HAMi repo link.

**Comment by wawa0210** [comment]
Thank you for your contribution
/lgtm
/approve

**Pull request merged**